CodePush.m

```diff
@@ -111,6 +117,20 @@ + (BOOL)isUsingTestConfiguration
     return testConfigurationFlag;
 }
 
+/*
+ * This returns the modified date as a string for a given file URL.
+ */
++ (NSString *)modifiedDateStringFromFileUrl:(NSURL *)fileUrl
```

Comment:
How about `modifiedDateOfFileAtURL`? We should definitely capitalize the `URL` part (in the parameter as well), and I feel like the "Of[X]At[Y]" format is more idiomatic of Cococa Touch APIs (e.g. `attributesOfItemAtPath`).